sdks/python/apache_beam/runners/dataflow/internal/dependency.py

```diff
@@ -565,7 +618,11 @@ def get_sdk_package_name():
     return BEAM_PACKAGE_NAME
 
 
-def _download_pypi_sdk_package(temp_dir):
+def _download_pypi_sdk_package(temp_dir, fetch_binary=False,
+                               language_version_tag='27',
```

Comment:
Except for the `fetch_binary` do you need the other args for now? For dataflow they should be fixed for the foreseeable future. Maybe the language version is an exception though.

Reply:
Sounds good.